samples/bluetooth/direct_test_mode/src/main.c

```diff
@@ -1,101 +1,33 @@
 /*
- * Copyright (c) 2020 Nordic Semiconductor ASA
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Not necessary - typically this header contains the original year of creation of the file.